### ISSUE TITLE
iOS: Mirroring Disconnected Offset Out of Range 

### ISSUE THREAD

**Issue by dhakshinamoorthys** [koush/vysor.io#856]
**To Reproduce**
Upgraded to latest Vysor-4.1.54(Runtime-4.1.29)
when you open the device, the offset out of range error will be displayed. "It must be 0>= and <=4 but received 8" 
device screen will not be displayed/mirrored

**Expected behavior**
The device screen should be visible

**Screenshots**
If applicable, add screenshots to help explain your [problem.](![Screenshot 2021-07-16 at 16 33 07](https://user-images.githubusercontent.com/35360533/125972664-a51a9161-25ea-468d-9c04-eb86f188a7f5.png))

**Desktop (please complete the following information):**
 - OS: macOS Catalina-10.15.7 
 
**iOS (please the following information):**
 - Device: iPhone6-12, iPad Pro and  iPadAir
 - OS: iOS 14.2
 - Vysor Bug Report Link: https://gist.github.com/VysorLogs/b56488cf7c0fd50e3b82ec1832034cbb

**iOS Bluetooth Issues (optional):**
 - Mac/Computer Model
 - Other devices connected with computer via Bluetooth
 - Other devices connected with iOS via Bluetooth

**Additional context**

**Comment by koush** [comment]
Thanks, will take a look. Are you mirroring multiple devices at once? Is this happening for all devices or just specific ones?

**Comment by koush** [comment]
I'm seeing a hard disconnect after the mirroring starts. Can you check if your iOS devices work when recording via QuickTime?

**Comment by dhakshinamoorthys** [comment]
No, I’m not trying to mirroring multiple devices at once. The app has been upgraded to latest version from that I’m not able to mirror any of the device.

**Comment by dhakshinamoorthys** [comment]
Could you please let me know the steps to test with QuickTime? I’m not aware of QuickTime. I would really appreciate your support
